docs/operational_guide/placement_changes.md

```diff
@@ -0,0 +1,132 @@
+# Placement Changes
+
+## Overview
+
+M3DB was designed from the ground up to a be a distributed / clustered database that is isolation group aware. Clusters will seamlessly scale with your data, and you can start with a cluster as small as 3 nodes and grow it to a size of several hundred nodes with no downtime or expensive migrations.
```

Comment:
nit with opening sentence: "to a be" -> "to be a" and "distributed / clustered database" -> "distributed (clustered)".

Perhaps:
```
M3DB was designed from the ground up to be a distributed (clustered) database that is zone or rack aware (by using isolation groups).
```